### PR TITLE
Create subsites-extras.md

### PR DESCRIPTION
**Pull request by aaronhirtenstein**
adding page for subsites extras in Content > Features section

### CODE REVIEW

**Comment by jpoesen** [comment]
Thanks @aaronhirtenstein. 

You mentioned in Slack that the page is not in a menu anywhere. Via the deploy preview I tried to deduce the URL of the page you added, but I'm not finding it.

Could you add a nav link and/or mention the path to the new page on the deploy preview so I can have a look?

**Review by markconroy — APPROVED** [review]

**Comment by jpoesen** [comment]
@markconroy I'm trying to understand the approval process here. How did you know which deploy preview page to review, and why did you choose to approve this when the new page is not reachable via any nav link (unless I'm missing something)?

**Comment by markconroy** [comment]
Sorry, I just read the content itself via the "changes" tab - https://github.com/localgovdrupal/docs/pull/283/files -  and that seemed good. I didn't check to see if it was in a menu.

**Comment by jpoesen** [comment]
Ok so the current docs approval process is checking the content files (markdown), not the final result. Understood.
Thanks.

Since the page is not reachable and needs nav, should this issue be set back to draft and reassigned to Aaron so he can add the nav where he intended it to be? Or is there a different workflow?

**Review by markconroy — CHANGES_REQUESTED** [review]
This needs a menu link before we can approve it.

**Comment by markconroy** [comment]
> Ok so the current docs approval process is checking the content files (markdown), not the final result.

I was too quick with my approval. We should be testing the final result. I hadn't thought of the need for a menu link.

I've set it back to "Changes requested".

**Comment by aaronhirtenstein** [comment]
Sorry for the confusion, both. And I messaged quickly before diving into a meeting all afternoon so had no time to respond. I'm afraid I don't know how to add a new commit to this, can you advise?

**Comment by aaronhirtenstein** [comment]
OK @jpoesen @markconroy I think I have sorted this now with an extra commit 🤞

File: docs/src/.vuepress/config.js [hunk 1]
```diff
@@ -223,6 +223,7 @@ export default defineUserConfig({
             '/content/features/workflow',
             '/content/features/elections',
             '/content/features/publications'
+            'content/features/subsites-extras'
```
Comment (jpoesen) [inline docs/src/.vuepress/config.js:226]: Needs leading slash.

Comment (aaronhirtenstein) [reply docs/src/.vuepress/config.js:226]: Doh! So it does!

File: docs/src/.vuepress/config.js [hunk 2]
```diff
@@ -223,6 +223,7 @@ export default defineUserConfig({
             '/content/features/workflow',
             '/content/features/elections',
             '/content/features/publications'
```
Comment (jpoesen) [inline docs/src/.vuepress/config.js:225]: Needs trailing comma.

**Comment by jpoesen** [comment]
Ok, got rid of the build error, but there are two problems:

nav link does appear in the sidebar, but as the path, not the title of the page:
image

nav link (`/content/features/subsites-extras.html`) --> 404

**Review by jpoesen — APPROVED** [review]

**Review by markconroy — APPROVED** [review]

**Comment by jpoesen** [comment]
Ok, so the problems were:
- previous menu item entry in config file needed trailing comma
- new menu item entry needed leading slash
- `subsites-extras.md` had a typo in the filename

**Pull request merged**